lib/spans/streaming-span-event.js

```diff
@@ -128,6 +128,12 @@ class StreamingSpanEvent {
 
     let span = null
     if (StreamingHttpSpanEvent.isHttpSegment(segment)) {
+      // Get external host from the segment name
+      const nameParams = segment.name.split('/')
```

Comment:
this seems heavy handed. what i would do is abstract away the assigning of the attributes for http externals, and just add another one there for host.  the reason i suggest abstracting is because we have multiple libraries that instrument libraries as http externals. in fact looking into this i think we were doing it wrong in grpc.  I can show you a diff of what I'm suggesting